bundles/org.openhab.voice.googlestt/src/main/java/org/openhab/voice/googlestt/internal/GoogleSTTService.java

```diff
@@ -157,6 +166,7 @@ public void abort() {
     }
 
     private void updateConfig() {
+        dispose();
```

Comment:
Perhaps it would be safer to explicitly `ungetOAuthService` here, so that this method will not free everything in case more cleanup is added to the `dispose` method.

Reply:
Yes, good remark.

Reply:
Changed